examples/system/cpp_pthread/main/cpp_pthread.cpp

```diff
@@ -0,0 +1,109 @@
+#include <iostream>
```

Comment:
Please add here the public domain/CC0 header found in other examples.

Reply:
Ofc.